frontend/src/components/PublishDashboardModal.tsx

```diff
@@ -95,7 +95,7 @@ function PublishDashboardModal(props: PathParams) {
                 }
             } catch (err) {
```

Comment:
err: any should do the same

Reply:
When adding `catch(err:any)` type scripts throws the error as follows:
```
TS1196: Catch clause variable cannot have a type annotation.